exercises/concept/savings-account/.meta/exemplar.rb

```diff
@@ -22,6 +22,8 @@ def self.annual_balance_update(balance)
   end
 
   def self.years_before_desired_balance(current_balance, desired_balance)
+    raise RuntimeError if current_balance.zero?
```

Comment:
If this gets added, then this exercise needs to rely on `exceptions` in the config.json. I think I'd also want a specific exception class etc, not just a RuntimeError.

Reply:
Is it OK if this exercise relies on Exceptions? Sorry for asking this, but this is my first PR on Exercism. 😄

Reply:
Exceptions are introduced in the concept exercise Simple Calculator and used in Moviegoer exercise.

It could be OK to use exceptions in this exercise, but we should put that as a prerequisite, if we do, as stated in [the above comment](https://github.com/exercism/ruby/pull/1346#discussion_r975232373).